extension/src/types.ts

```diff
@@ -85,3 +85,64 @@ export enum MessageType {
     DEBUG_MESSAGE,
     CHANGE_ROOM,
 }
+
+/**
+ * Video status for one user
+ */
+enum UserVideoStatus {
+    Unknown, // Still Loading of DOM, or not on the video URL, or any othe reason (no info from Video API)
+    Buffering, // has not reached minBufferLength (video is paused or not started)
+    Playing, // is currently playing the video
+    Ready, // (equivalent of Waiting or Paused) has buffered enough of the video, is pending start signal
+}
+
+/**
+ * Video status for all players that have completed initial Sync
+ */
+enum VideoSyncStatus {
+    Paused, // video is paused and no action should be taken (apart from buffering)
```

Comment:
In TS an enum will use value like 0, 1, 2 and match them for us. For debugging purpose it is better to write them like this

```ts
enum VideoSyncStatus {
  PAUSED = "PAUSED";
```